### ISSUE TITLE
copyright symbol must have trailing whitespace

### ISSUE THREAD

**Issue by stevejpurves** [executablebooks/mystjs#199]
### What version of `mystjs` are you using?

0.1.11

### How did you install myst?

npm

### What operating system are you using?

Mac

### Which area is this feature request for?

Export: LaTeX or PDF

### Describe the Bug

MyST content containing a copyright symbol like:

```
Promis©Someone
```

Will be rendered to:

```
Promis\textcopyrightSomeone
```

Resulting in a latex build error as `\textcopyrightBelux` is considered a command.

### Expected Behavior

 The Correct sequence must include a space after the copyright command:
 ```
 Promis\textcopyright Someone
 ```

**Comment by rowanc1** [comment]
https://myst-tools.org/sandbox?tab=latex&myst=UAByAG8AbQBpAHMAqQBTAG8AbQBlAG8AbgBlAA%3D%3D

**Comment by stevejpurves** [comment]
@fwkoch hitting this one again on recent exports.

**Comment by rowanc1** [comment]
@stevejpurves can you check if this fixes your problem?

- #323

**Comment by rowanc1** [comment]
@stevejpurves this should be mostly fixed in #323.

There is still a typography bug in PDF:
![image](https://user-images.githubusercontent.com/913249/228022761-d8d12cb0-8c56-4d5d-8985-bd64b14e3972.png)

But this should at least get us a build. And we can open up another issue for tracking spaces after text symbols.
